.github/workflows/docker_image.yml

```diff
@@ -0,0 +1,67 @@
+name: Push docker images to the GitHub registry
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+  workflow_dispatch:
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build_push:
+    runs-on: ubuntu-latest
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          driver-opts: network=host
+      - name: Checkout Apache Buildr
+        uses: actions/checkout@v3
+        with:
+          repository: sathwik/docker-buildr
+          path: apache-buildr
+      - name: Fix dockerfile
```

Comment:
Why exactly do we need this?

Reply:
The Dockerfile tries to install the newest version of `bundler` which is incompatible with some other dependencies, that's why we have to change the Dockerfile to install the newest compatible version which is 2.3.26 to fix this error:

```
ERROR:  Error installing bundler:
The last version of bundler (>= 0) to support your Ruby & RubyGems was 2.3.26. Try installing it with `gem install bundler -v 2.3.26`
bundler requires Ruby version >= 2.6.0. The current ruby version is 2.3.0.
```

Reply:
Because the Dockerfile is downloaded from their repository and this way we don't have to worry about the license.

Reply:
Can´t we use another image as builder then? Looks like a very hacky solution atm

Reply:
I haven't found another image with Apache Buildr and Jruby. I think the cleanest solution would be to include the Dockerfile after all with the modification and the appropriate license statement.